GLTFSDK.Test/Source/GLBResourceWriterTests.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "stdafx.h"
+
+#include <GLTFSDK/GLBResourceWriter.h>
+
+#include "TestUtils.h"
+
+using namespace glTF::UnitTest;
+
+namespace Microsoft
+{
+    namespace  glTF
+    {
+        namespace Test
+        {
+            GLTFSDK_TEST_CLASS(GLBResourceWriterTests)
+            {
+                GLTFSDK_TEST_METHOD(GLBResourceWriterTests, WriteBufferView_Empty)
```

Comment:
Test needs to read the output and confirm that it's a valid GLB by parsing it again.